### PR TITLE
Adding new categories

### PR DESCRIPTION
**Pull request by Isaac-El**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added new category's (a parameter of integration) options

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cyberpion/pack_metadata.json [hunk 1]
```diff
@@ -17,5 +17,14 @@
...
         "marketplacev2"
+    ],
+    "dependencies": {
+        "CoreAlertFields": {
+            "mandatory": true,
+            "display_name": "Core Alert Fields"
+        }
+    },
+    "displayedImages": [
+        "CoreAlertFields"
```
Comment (yaakovi) [inline Packs/Cyberpion/pack_metadata.json:28]: Why did you add this change?

File: Packs/Cyberpion/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cyberpion
+- Added more incidents categories
```
Comment (yaakovi) [inline Packs/Cyberpion/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added support for the following incidents categories:
  - Network
  - Web
  - TLS
  - Email Server
  - Mobile
```

**Review by yaakovi — APPROVED** [review]

File: Packs/Cyberpion/pack_metadata.json (hunk 1 above)
Comment (Isaac-El) [reply Packs/Cyberpion/pack_metadata.json:28]: It was generated automatically by demisto-sdk commands. You can remove it if you want (I don't understand the meaning of it).
